machine/v1beta1/types_vsphereprovider.go

```diff
@@ -190,6 +190,9 @@ type Workspace struct {
 	// ResourcePool is the resource pool in which VMs are created/located.
 	// +optional
 	ResourcePool string `gcfg:"resourcepool-path,omitempty" json:"resourcePool,omitempty"`
+	// VMGroup is the cluster vm group in which virtual machines will be added for vm host group based zonal.
+	// +optional
+	VMGroup string `gcfg:"vmGroup,omitempty" json:"vmGroup,omitempty"`
```

Comment:
this appears to be missing a featuregate.  Can you add the appropriate gate that allows us to ensure that all testing is complete before the API is exposed.

Reply:
The feature gate is defined in this PR: https://github.com/openshift/api/pull/1999

Reply:
unless what you meant was to add
```
// +openshift:validation:featureGate=VSphereHostVMGroupZonal
```
?

Reply:
Add the marker for the gate please, it won't actually do anything as this is a raw extension but lets add so we remember what's gating this

In the MAPI webhook we should make sure that this field is not allowed when the gate is diabled